storage/postgres/notificator.go

```diff
@@ -323,40 +365,31 @@ func (n *Notificator) getRecipients(platformID string) []*types.Platform {
 func (n *Notificator) sendNotificationToPlatformConsumers(platformConsumers []storage.NotificationQueue, notification *types.Notification) {
 	for _, consumer := range platformConsumers {
 		if err := consumer.Enqueue(notification); err != nil {
-			log.C(n.ctx).WithError(err).Infof("consumer %s notification queue returned error %v", consumer.ID(), err)
+			log.C(n.ctx).WithError(err).Infof("Consumer %s notification queue returned error %v", consumer.ID(), err)
 			consumer.Close()
 		}
 	}
 }
 
 func (n *Notificator) stopConnection() {
-	err := n.stopListening()
-	logger := log.C(n.ctx)
-	if err != nil {
-		logger.WithError(err).Info("could not unlisten notification channel")
-	}
+	n.stopProcessing()
 	n.connectionMutex.Lock()
 	defer n.connectionMutex.Unlock()
-	if err = n.connection.Close(); err != nil {
-		logger.WithError(err).Info("could not close db connection")
-	}
-}
 
-func (n *Notificator) stopListening() error {
-	n.connectionMutex.Lock()
-	defer n.connectionMutex.Unlock()
-	if atomic.LoadInt32(&n.isListening) == aFalse {
-		return nil
+	atomic.StoreInt32(&n.isConnected, aFalse)
```

Comment:
can it happen that `connection.Close` fails but `isConnected` is `true`? as this is called only when SM is stopped I suppose this won't be a problem, but still